examples/import-in-node/src/index.ts

```diff
@@ -0,0 +1,92 @@
+import * as dotenv from "dotenv";
+import * as path from "path";
+import { input } from "@inquirer/prompts";
+import { Turnkey } from "@turnkey/sdk-server";
+import { Crypto } from "@peculiar/webcrypto";
+import { encryptPrivateKeyBundle, encryptWalletBundle } from "@turnkey/crypto";
```

Comment:
`encryptPrivateKeyBundle` and `encryptWalletBundle` suggest that we have a private key or wallet bundle, and that we are encrypting them. This is not what's happening. We are actually using a bundle to encrypt a piece of data. So I think the "with" or "via" or "to" in the name is needed (`encryptPrivateKeyWithBundle` or `encryptPrivateKeyViaBundle` or `encryptPrivateKeyToBundle`)